core/src/View/MDSpan/Kokkos_MDSpan_Accessor.hpp

```diff
@@ -475,12 +479,23 @@ template <class ElementType, class MemorySpace,
 using CheckedRelaxedAtomicAccessor =
     SpaceAwareAccessor<MemorySpace, AtomicAccessorRelaxed<ElementType>>;
 
+template <class ElementType, class MemorySpace,
+          class MemoryScope = desul::MemoryScopeDevice>
+using CheckedRestrictAccessor =
+    SpaceAwareAccessor<MemorySpace, RestrictAccessor<ElementType>>;
+
 template <class ElementType, class MemorySpace,
           class MemoryScope = desul::MemoryScopeDevice>
 using CheckedReferenceCountedRelaxedAtomicAccessor = SpaceAwareAccessor<
     MemorySpace, ReferenceCountedAccessor<ElementType, MemorySpace,
                                           AtomicAccessorRelaxed<ElementType>>>;
 
+template <class ElementType, class MemorySpace>
+using CheckedReferenceCountedRestrictAccessor =
+    SpaceAwareAccessor<MemorySpace,
+                       ReferenceCountedAccessor<ElementType, MemorySpace,
+                                                RestrictAccessor<ElementType>>>;
```

Comment:
None of these aliases are used anymore.

Reply:
They were used in the unit tests so I just moved them over there.